src/plugins/es_ui_shared/server/license/license.ts

```diff
@@ -0,0 +1,118 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { i18n } from '@kbn/i18n';
+import {
+  Logger,
+  KibanaRequest,
+  KibanaResponseFactory,
+  RequestHandler,
+  RequestHandlerContext,
+} from 'src/core/server';
+
+/* eslint-disable @kbn/eslint/no-restricted-paths */
```

Comment:
@elastic/kibana-core Do we still need this linting rule?

Reply:
For concrete imports, we definitely do, as it would plainly breaks on OSS distrib.

For types, it's not a 'technical' requirement, but until the basic plugins are moved out of xpack, we should stick with that rule.

Reply:
AFAIK we don't release OSS distributable starting from v7.11 https://www.elastic.co/downloads/past-releases#kibana-oss

But I'd rather plugins didn't import from `x-pack`. Note that the code in `src/` and `x-pack` have different licenses, so it's easy to leak commercial code outside by importing `runtime` code accidentally.
Regarding the current use case: why `licensing` code lives in `src/` at all? `Licensing` compatible types can be declared in the current file, but maybe an `x-pack` plugin is a better place?